src/test/java/org/mifos/integrationtest/config/MockServerConfig.java

```diff
@@ -23,5 +23,10 @@ public WireMockServer getMockServer() {
         return MockServerConfig.singleInstance;
     }
 
+    @Override
+    public String getBaseUri() {
+        return "http://localhost:"+getMockServer().port();
```

Comment:
can we use mock server at localhost ?

Reply:
yes

Reply:
what is the use of adding a server in localhost? what are we trying to test here?

Reply:
testing if mockserver is able to start a test or not.